legal-api/src/legal_api/services/filings/validations/restoration.py

```diff
@@ -40,7 +40,8 @@ def validate(business: Business, restoration: Dict) -> Optional[Error]:
     elif restoration_type in ('fullRestoration', 'limitedRestorationToFull'):
         msg.extend(validate_relationship(restoration))
 
-    msg.extend(validate_name_request(restoration, business.legal_type, filing_type))
+    if restoration.get('filing', {}).get('restoration', {}).get('nameRequest', {}).get('nrNumber', None):
```

Comment:
Are there tests for the 3 cases?

Reply:
no, name requests aren't checked currently for restoration validation tests

Reply:
That would be good to do, since (1) the business name is required, and (2) we would catch it if a future change breaks something.

Reply:
ok, will add some tests to restoration validator tests for NR block.

Reply:
Thanks. Looks good!